pkg/nad/list.go

```diff
@@ -0,0 +1,56 @@
+package nad
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	nadV1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	goclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// List returns NADs inventory in the given namespace.
+func List(apiClient *clients.Settings, nsname string) ([]*Builder, error) {
+	if apiClient == nil {
+		glog.V(100).Infof("The apiClient is empty")
+
+		return nil, fmt.Errorf("nadList 'apiClient' cannot be empty")
+	}
+
```

Comment:
could you please attach the scheme here like in the [Pull function](https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/nad/builder.go#L89-L94)